test.js

```diff
@@ -130,16 +190,16 @@ test('stripFinalNewline: false', async t => {
 	t.is(stdout, 'foo\n');
 });
 
-test('stripFinalNewline on failure', async t => {
-	const {stderr} = await t.throwsAsync(execa('noop-throw', ['foo'], {stripFinalNewline: true}));
-	t.is(stderr, 'foo');
-});
-
 test('stripFinalNewline in sync mode', t => {
 	const {stdout} = execa.sync('noop', ['foo'], {stripFinalNewline: true});
 	t.is(stdout, 'foo');
 });
 
+test('stripFinalNewline on failure', async t => {
+	const {stderr} = await t.throwsAsync(execa('noop-throw', ['foo'], {stripFinalNewline: true}));
+	t.is(stderr, 'foo');
+});
```

Comment:
This test should be moved back in the same position as it were.

Reply:
The reason why the change is visible is because I screwed up with the rebase. I think I'll attempt a second rebase on the current master as there are newer changes anyway.

Reply:
You don't need to. Just move the test and commit it, and it should remove that diff.